### PR TITLE
updated all titles, changed HTML, fixed assoc tests

### PR DESCRIPTION
**Pull request by dinophile**
Updated things and stuff and tested and brewed iced tea while watching Downton Abbey.

### CODE REVIEW

File: locales/en.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "404": "404",
   "500": "500",
-  "app.title": "Generate privacy and consent forms",
+  "app.title": "Bananas!! Generate privacy and consent forms",
```
Comment (dinophile) [inline locales/en.json:4]: I like oranges not bananas!

Comment (dinophile) [inline locales/en.json:4]: Add guavas too!

Comment (caitlinmtom) [inline locales/en.json:4]: This is another comment.

**Review by kaira5 — CHANGES_REQUESTED** [review]
Test

**Review by caitlinmtom — CHANGES_REQUESTED** [review]
Please change some things.

File: locales/en.json (hunk 1 above)
Comment (jeanafrost) [inline locales/en.json:4]: Hi banana!

**Review by caitlinmtom — APPROVED** [review]

**Review by kaira5 — APPROVED** [review]

**Review by jeanafrost — APPROVED** [review]
👯

**Pull request closed without merging**